ompi/communicator/comm.c

```diff
@@ -1470,7 +1470,7 @@ int ompi_intercomm_create_from_groups (ompi_group_t *local_group, int local_lead
     *newintercomm = MPI_COMM_NULL;
 
     /* create a local communicator first. create a unique tag for this communicator */
-    asprintf (&sub_tag, "%s-OMPIi-%s", tag, OPAL_NAME_PRINT(ompi_group_get_proc_name (local_group, local_leader)));
+    (void)asprintf (&sub_tag, "%s-OMPIi-%s", tag, OPAL_NAME_PRINT(ompi_group_get_proc_name (local_group, local_leader)));
```

Comment:
could this be opal_asprintf?

Reply:
Could and should.  Casting the return of the function to void won't get rid of the unused return warning on modern GCC.

Reply:
fixed.